Source/WebKit/UIProcess/WebAuthentication/Cocoa/HidService.mm

```diff
@@ -30,6 +30,8 @@
 
 #import "CtapHidDriver.h"
 #import "HidConnection.h"
+#import <IOKit/hid/IOHIDDevice.h>
+#import <wtf/cf/TypeCastsCF.h>
```

Comment:
Is this a non-unified build finding? It's fine to keep it in, but doesn't seem relevant to the patch unless you discovered a build failure on some platforms.